src/main/java/org/reaktivity/nukleus/http2/internal/routable/stream/TargetOutputEstablishedStreamFactory.java

```diff
@@ -161,57 +151,43 @@ private void afterBeginOrData(
             }
         }
 
-        private void afterEnd(
-            int msgTypeId,
-            DirectBuffer buffer,
-            int index,
-            int length)
+        private void afterEnd(int msgTypeId, DirectBuffer buffer, int index, int length)
         {
             processUnexpected(buffer, index, length);
         }
 
-        private void afterRejectOrReset(
-            int msgTypeId,
-            MutableDirectBuffer buffer,
-            int index,
-            int length)
+        private void afterRejectOrReset(int msgTypeId, MutableDirectBuffer buffer, int index, int length)
         {
             if (msgTypeId == DataFW.TYPE_ID)
             {
                 dataRO.wrap(buffer, index, index + length);
-                final long streamId = dataRO.streamId();
+                long streamId = dataRO.streamId();
```

Comment:
It feels like some code checker is driving you to remove these `final` modifiers, is that correct?

We've been using `final` modifier to clearly communicate that the variable is not modifiable.